### DISCUSSION TITLE
Instructions for setting up multiple relay servers

### DISCUSSION THREAD

**Discussion by d0gman** [rustdesk/rustdesk-server-pro#324] [General]
Can the official provide a complete detailed instruction document for building multiple relay servers in pro? After purchasing it, I have been unsuccessful in building it.

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/relay/

**Reply by d0gman** [reply]
After I add them to the relay, these keys are random. How do I add them to the relay to make them effective?
